server/queue/queue-rabbitmq/src/main/java/org/apache/james/queue/rabbitmq/view/cassandra/configuration/CassandraMailQueueViewConfiguration.java

```diff
@@ -88,4 +89,21 @@ public int getBucketCount() {
     public Duration getSliceWindow() {
         return sliceWindow;
     }
+
+    @Override
+    public final boolean equals(Object o) {
```

Comment:
sorry but if I can't trust a commit comment, I'm not sure I'll ever approve this PR.
Please be careful, when a comment says "this is a file move" to not change any line of code unrelated to that move.

Reply:
Fixed with the rebase.

I did not had a deep look in this commit taken from https://github.com/linagora/james-project/pull/1748, sorry